client/src/components/About/AboutInformation/AboutInformation.js

```diff
@@ -53,35 +60,45 @@ export default class AboutInformation extends React.Component {
                 <h1>
                     About Monuments + Memorials
                 </h1>
-                {/* TODO: Replace click here with link to sign-in page */}
                 <p>
                     <span className="font-italic">Monuments + Memorials</span> is a crowd-sourced initiative developed at
                     the Rochester Institute of Technology under the direction of Dr. Juilee Decker. Students in her spring
-                    2019 history course <span className="font-italic">Monuments & Memory</span> began data collection by
-                    examining 26 states or territories in the US. The following fall, software engineering students in the
+                    2019 history course <span className="font-italic">Monuments + Memory</span> began data collection by
+                    examining 26 states or territories of the US. The following fall, software engineering students in the
                     Golisano College of Computer and Information Sciences built this web application to house data and to
-                    generate further interest and data collection. To view monuments and memorials, {mapNavLink}. To enter
-                    data, click here.
+                    generate further interest and data collection. The spring 2020 history course added to the existing
+                    data by contributing 900 additional monuments + memorials, so that all states are represented, as
+                    are two territories (Guam and Puerto Rico).
+                </p>
+                {/* TODO: Replace click here with link to sign-in page */}
+                <p>
+                    Our goal is to document all temporary and permanent monuments + memorials throughout the United
+                    States and territories. We are looking for contributors to add information, please join us by
+                    creating an account (sign-in link here) or by contacting the organizers
+                    &nbsp;<span className="email-link">contact@monuments.us.org</span>. To view monuments and memorials,
```

Comment:
Is there a better way to do links to emails? I just added some styling to it to make it look like a link but you can't actually click it, which may be a bad UI.

Reply:
Actually, just noticed how @benvogler did email links on the Manage Users Page. I changed how I was doing them to do the same.